src/couch/src/couch_att.erl

```diff
@@ -178,6 +178,9 @@
 
 -type att() :: #att{} | attachment() | disk_att().
 
+% 1024 * 1024 * 1024
+-define(GB, 1073741824).
```

Comment:
better to say `-define(GB, (1024*1024*1024)).` I'm sure the compiler will do the right hing.

Reply:
Done in 83be3921b